new-issue/command.py

```diff
@@ -0,0 +1,132 @@
+import sys
+import os
+import json
+
+from devchat.llm.chat import chat_json
+from github import Github
+from github import Auth
+# from common_util import editor
+
+def read_config():
+  config_path = os.path.join(os.getcwd(), '.devchat', 'information', 'config', 'config.json')
+  if not os.path.exists(config_path):
+    print('Config file does not exist')
+    return None
+  with open(config_path, 'r') as f:
+    return json.load(f)
+
+def write_config(repo_path, github_token):
+  config_path = os.path.join(os.getcwd(), '.devchat', 'information', 'config', 'config.json')
+  with open(config_path, 'w') as f:
+    json.dump({'repo_path': repo_path, 'github_token': github_token}, f)
+
+PROMPT= prompt = """
+pls return a JSON object without line break(the response must be parsed as json, not contain any redundant chars), including a string field Title and a markdown string field called Body.
+{information}
+{request}
+{context}
+"""
+@chat_json(prompt=PROMPT)
+def get_issue_content(information, request, context):
+  pass
+
+PROMPT= prompt = """
+I will request you to write an issue. The request is consists of two parts, Firstly, I will give a context list(organized by concept name). You may choose most related ones and return it as a json object(with key files) according to the requst content. Then I will ask you for issue content with the context you chosen. Now is the first request, pls only return an json object with context files you chosen. context files: {context_list}, issue content:
+`{request_content}`
+"""
+@chat_json(prompt=PROMPT)
+def get_related_context_file(context_list, request_content):
+  pass
+
+# @editor("Please specify the issue's repository, "
+#         "If the issue is within this repository, no need to specify. "
+#         "Otherwise, format as: username/repository-name")
+# @editor("Input your github TOKEN to access github api:")
+# def set_config(repo_path, github_token):
+#   pass
+
+def get_request(path):
+  if not os.path.exists(path):
+      return
+  with open(path, 'r') as f:
+    request = f.read()
+  return request
+
+def read_api_config():
+  config = read_config()
+  # set config by devlake ui
+  # repo_path = ''
+  # github_token = ''
+  # if config != None:
+  #   repo_path = config['repo_path']
+  #   github_token = config['github_token']
+  # else:
+  #   # request user config
+  #   repo_path, github_token = set_config(repo_path, github_token)
+
+  # write_config(repo_path, github_token)
+  repo_path = config['repo_path']
+  github_token = config['github_token']
+  return repo_path, github_token
+def create_github_issue(github_token, repo_path, issue_content, issue_title):
+  auth = Auth.Token(github_token)
+  g = Github(auth=auth)
+  repo = g.get_repo(repo_path)
+  issue = repo.create_issue(title=issue_title, body=issue_content)
+  print(issue, flush=True)
+
+def main():
+  # get user input
+  user_input = sys.argv[1]
+  params = user_input.strip().split(' ')
```

Comment:
Bug: if your provide parameter with english, the request context will be seperated by space.